apps/docs/components/Header.tsx

```diff
@@ -70,7 +70,7 @@ export function SectionLinks({ sectionId }: { sectionId?: string | null }) {
 				Reference
 			</a>
 			<a
-				href="/examples/basic/develop"
+				href="/examples/basic/basic"
```

Comment:
renamed this because we probably want to use the fullscreen example for development now